### DISCUSSION TITLE
X64编译Turboacc的时候是否应该勾选shortcutFE？

### DISCUSSION THREAD

**Discussion by zzz6839** [immortalwrt/immortalwrt#706] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/57741549/172141058-c1e971e1-ae1a-476e-88e7-2fbe2790e030.png)

我的软路由是J1900，在编译的时候是否需要勾选这个选项感到困惑。查了些资料，我的理解是shortcutFE和flow offload一个是高通的，一个是openwrt集成的，两个都是软件，但是flow offload一直有在开发维护支持更多设备，但是SFE最适合高通的芯片，所以不需要考虑SFE，是这样吗？

**Comment by 1715173329** [comment, marked as answer]
SFE 几乎是为 QSDK 高通内核定制的东西，搬到主线各种爆炸，包括造成 kernel panic 以及 ~33% 的 udp packet loss。
默认 flow offloading 即可。

**Reply by zzz6839** [reply]
3Q
